SKILL.tsv

```diff
@@ -4048,181 +4048,181 @@ SKILL_20150428_004047	{memo X}Summon Demon type boss{nl}Demon power +#{CaptionRa
+SKILL_20150428_004057	$Evasion -#{CaptionRatio}#{nl}Pointing AI Duration: 90 seconds
+SKILL_20150428_004058	$Detects hidden monsters {nl}Hounding AI Duration: 90 seconds
+SKILL_20150428_004059	$Consumes #{SpendPoison}# Poison Pot Poison
 SKILL_20150428_004060	{memo X}Attack #{SkillAtkAdd}#{nl}Duration 30 seconds
 SKILL_20150428_004061	Throw tear gas on target area. Target falls under Darkness.
-SKILL_20150428_004062	Tear gas duration #{CaptionTime}# seconds
+SKILL_20150428_004062	$Tear Gas Duration: #{CaptionTime}# seconds
 SKILL_20150428_004063	{memo X}Attack #{SkillAtkAdd}#{nl}Strike 2 times{nl}Critical rate #{CaptionRatio}#%
 SKILL_20150428_004064	{memo X}Attack #{SkillAtkAdd}#{nl}Bleed Duration #{CaptionTime}# seconds{nl}Use 1 Broadhead Arrow
 SKILL_20150428_004065	{memo X}Attack #{SkillAtkAdd}#{nl}Physical Defense -#{CaptionRatio}#% {nl}Use 1 Bodkin Arrow
 SKILL_20150428_004066	{memo X}Attack #{SkillAtkAdd}#{nl}Duration 10 seconds{nl}Use 1 Magic Arrow
 SKILL_20150428_004067	{memo X}Attack #{SkillAtkAdd}#{nl}Duration 10 seconds{nl}Use 1 Gather Corpse
-SKILL_20150428_004068	Hanging Shot
-SKILL_20150428_004069	Patron Saint Duration #{CaptionTime}# seconds{nl}Trigger rate 50%{nl}Trigger #{CaptionRatio}# times
+SKILL_20150428_004068	$Hanging Shot
+SKILL_20150428_004069	$Guardian Saint Duration: #{CaptionTime}# seconds{nl}Chance on hit: 50%{nl}Charges: #{CaptionRatio}#
 SKILL_20150428_004070	$Creates a magic circle in front of you that increases the DEX of allies and decreases the enemy's evasion and physical defense. Party members will receive the effect without stepping on the magic circle.
 SKILL_20150428_004071	$Gives a blessing providing additional damage to the attacks of your party members and allies.
-SKILL_20150428_004072	Additional damage +#{CaptionRatio}#{nl}Attack #{CaptionRatio2}# times{nl}Target allies #{CaptionTime}#{nl}Duration 45 seconds{nl}Use 1 Holy Powder
+SKILL_20150428_004072	$Additional Damage +#{CaptionRatio}#{nl}Number of hits: #{CaptionRatio2}#{nl}Target allies #{CaptionTime}#{nl}Duration: 45 seconds{nl}Consumes 1 Holy Powder
 SKILL_20150428_004073	{memo X}Body's moving range #{CaptionRatio}#{nl}Body's attack +#{CaptionRatio2}#
 SKILL_20150428_004074	{memo X}Reflect as much as target's Physical attack{nl}Additional damage reflected #{SkillAtkAdd}#{nl}Max. duration #{CaptionTime}# seconds{nl}Range projectile, Magic defense not possible
 SKILL_20150428_004075	{memo X}Attack #{SkillAtkAdd}#{nl}Use 2 Stamina
 SKILL_20150428_004076	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Attack enemy with tip of the hand to cause internal damage.
 SKILL_20150428_004077	{memo X}Attack per strike #{SkillAtkAdd}#{nl}Use #{CaptionRatio}# SP per 0.5 seconds
-SKILL_20150428_004078	Max. duration #{CaptionTime}# seconds{nl}Range projectile, Magic defense possible
-SKILL_20150428_004079	Max. purchase qty. #{CaptionRatio}#
+SKILL_20150428_004078	$Maximum Duration: #{CaptionTime}# seconds{nl}Able to defend from ranged projectiles and magic
```

Comment:
We should probably switch the sentences, with the explanation coming first, and the key: value pairs afterwards.